platform/jewel/markdown/int-ui-standalone-styling/intellij.platform.jewel.markdown.intUiStandaloneStyling.iml

```diff
@@ -40,6 +38,7 @@
     <orderEntry type="module" module-name="intellij.platform.jewel.intUi.standalone" />
     <orderEntry type="module" module-name="intellij.platform.jewel.markdown.extensions.gfmAlerts" />
     <orderEntry type="module" module-name="intellij.platform.jewel.markdown.extensions.gfmTables" />
+    <orderEntry type="module" module-name="intellij.platform.jewel.markdown.extensions.images" />
```

Comment:
I just realised that these should be marked as provided dependencies, not as compiled (see [this](https://www.jetbrains.com/help/idea/working-with-module-dependencies.html#z5dxrlw_51) for info). We should not add all the dependencies as the equivalent of `implementation` for Gradle, they should be `compileOnly`.

Same in the bridge...